### ISSUE TITLE
export code in pytorch

### ISSUE THREAD

**Issue by sky1ove** [viscom-ulm/Net2Vis#24]
Two things can be improved:

1. It could be very useful if the input Keras modeling code could be not only visualized, but also be exported in pytorch, as many people are using pytorch
2. Another thing that can be added is that pytorch modeling code visualization, and the corresponding export could be in Keras.

**Comment by Sparkier** [comment]
Thank you for the kind words :) Glad you enjoy Net2Vis.

I plan on adding the same functionality for PyTorch models, but I did not get to implement that yet.

As for the code conversion, I am not sure if that is something we can/should provide in here. As far as I know conversion between Kara's and PyTorch is not always trivial, and might introduce errors. If there is an easy way or a package to do this, I might change my mind, but I don't think this is closely related to the value of Net2Vis.

**Comment by sky1ove** [comment]
I agree with you that code conversion is kinda irrelevant with Net2Vis. But
making a pytorch model visualization package similar to Net2Vis could be
fun! A lot of current visualizers for models just use the same shape for
layers ( like netron) with layer information, which makes no much
difference with reading the code. Net2Vis using different shapes based on
the layer size is really fantastic.
